### PR TITLE
Add wachy - userspace eBPF perf debugging tool

### PR DESCRIPTION
**Pull request by viveksjain**
Let me know if you have any comments!

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -242,6 +242,7 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [rbpf](https://github.com/qmonnet/rbpf) - Written in Rust. Interpreter for Linux, macOS and Windows, and JIT-compiler for x86_64 under Linux.
 - [PREVAIL](https://github.com/vbpf/ebpf-verifier) - A user space verifier for eBPF [using an abstract interpretation layer](https://elazarg.github.io/pldi19main-final.pdf), with support for loops.
 - [oster](https://github.com/grantseltzer/oster) - Written in Go. A tool for tracing execution of Go programs by attaching eBPF to uprobes.
+- [wachy](https://rubrikinc.github.io/wachy/) - A tracing profiler that aims to make eBPF uprobe-based debugging easier to use. This is done by connecting it back to the source code and providing a UI for interactive drilldown analysis.
```
Comment (qmonnet) [inline readme.md:245]: > This is done by connecting it back to the source code

Not clear to me what this means, I don't believe this means instrumenting the code, right? You mean displaying the code in the UI, a bit like when annotating with `perf`?

**Review by qmonnet — COMMENTED** [review]
Thanks for the contribution! Just one question below.

**Comment by viveksjain** [comment]
Thanks for the comment, hopefully the update is clearer

**Review by qmonnet — APPROVED** [review]
Yes, this is clearer now, thanks again!

**Pull request merged**